x-pack/plugins/enterprise_search/public/applications/app_search/components/role_mappings/role_mappings_logic.test.ts

```diff
@@ -0,0 +1,437 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockFlashMessageHelpers, mockHttpValues, mockKibanaValues } from '../../../__mocks__';
+import { LogicMounter } from '../../../__mocks__/kea.mock';
+
+import { engines } from '../../__mocks__/engines.mock';
+
+import { nextTick } from '@kbn/test/jest';
+
+import { asRoleMapping } from '../../../shared/role_mapping/__mocks__/roles';
+import { ANY_AUTH_PROVIDER } from '../../../shared/role_mapping/constants';
+
+import { RoleMappingsLogic } from './role_mappings_logic';
+
+describe('RoleMappingsLogic', () => {
```

Comment:
@JasonStoltz I did a brief skim of this file but didn't examine it too closely - any chance I could tag you in to review this test file Monday morning (since you rock at these)? 🙇‍♀️